aiida/orm/implementation/general/node.py

```diff
@@ -1708,8 +1708,10 @@ def _store_from_cache(self, cache_node, with_transaction):
             if key != Sealable.SEALED_KEY:
                 self._set_attr(key, value)
 
-        for path in cache_node.get_folder_list():
-            self.add_path(cache_node.get_abs_path(path), path)
+        self.folder.replace_with_folder(
+            cache_node.folder.get_abs_path(''),
+            overwrite=True
```

Comment:
Can you explicitly set move=False? Just in case we change inadvertently the default. 
also, in the test above
1. just as a note, raw_input will be there only for JobCalculations
2. maybe you can check that 'raw_input' is still there also in the node you cached from, to avoid that by mistake we remove the files?
Thanks!

Reply:
Yeah, makes sense.

1. I knew that, but put it explicitly in the test now.
2. Also done.

I guess ``Node.copy`` was already broken for a while, but wasn't really used for much.

Cheers!

Reply:
FYI: Node.copy, now removed from develop, was used to create a "copy" of the calculation, to possibly modify the inputs and resubmit - so it was intended that it didn't copy the raw_inputs.

Reply:
Ah I see.. well then I guess I saw that method and used it in a wrong way :slightly_smiling_face:

Reply:
Wait, doesn't that mean #1746 is wrong because it changes the behavior of ``copy``?

Reply:
ops... yes! I didn't realize there you changed copy, I thought it was the implementation of caching only... could you fix?